gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/UserServiceImpl.java

```diff
@@ -848,6 +848,15 @@ public UserEntity create(ExecutionContext executionContext, final NewPreRegister
         );
     }
 
+    private boolean isServiceAccount(User user) {
```

Comment:
Maybe we can move this method directly in User class ?

Reply:
Honestly, I don't know if that's worth it.
I'm really unsure why we went that way to manage service accounts, instead of just storing a boolean within the user entity...

Reply:
🤷